recipes/cad1/task1/baseline/merche_batches_results.py

```diff
@@ -0,0 +1,25 @@
+"""Join batches scores into a single file."""
+
+import hydra
+import pandas as pd
+from omegaconf import DictConfig
+
+
+@hydra.main(config_path="", config_name="config")
+def join_batches(config: DictConfig) -> None:
+    """
+    Join batches scores into a single file.
+
+    """
+    batches_results = []
+    for batch in range(config.evaluate.batch_size):
+        batches_results.append(
+            pd.read_csv(f"scores_{batch + 1}-{config.evaluate.batch_size}.csv")
+        )
+    df_res = pd.concat(batches_results, ignore_index=True)
+    df_res.to_csv("scores.csv", index=False)
```

Comment:
Would it perhaps be useful to have an additional argument `outpath: Union[str, Path] = "./"` and make the output location configurable? I'm not sure how/where people use this but making such things flexible helps users.

Would also require...

```python
    outpath = Path(outpath)
```

...to ensure the object is of type `Path()` and could then...

```suggestion
    df_res.to_csv(outpath / "scores.csv", index=False)
```

Reply:
The output directory is already defined in `path.exp_folder` in the config.
So, the combined file will be stored next to the batches results in the experiment directory.
That is one of the beauties of Hydra, you can define the experiment path and that path is then implicit throughout the whole experiment.